src/plans/alternatig_gradient_plan.jl

```diff
@@ -0,0 +1,287 @@
+@doc raw"""
```

Comment:
There is a typo in the name of this file (alternatig instead of alternating).

Reply:
Oh, thanks, currently I seem to have even introduced a bug, the third example currently does not converge anymore.